src/common/path/platform/c++17/path.cpp

```diff
@@ -97,7 +97,7 @@ error::Error Permissions(const string &file_path, const vector<Perms> perms) {
 	if (perms.size() == 0) {
 		return error::NoError;
 	}
-	fs::perms p;
+	fs::perms p {};
```

Comment:
Is there a difference?

Reply:
Was just CI complaining about it. This fixed it...

So I'm guessing a false positive? :shrug:

Reply:
Hmm, probably. I think theoretically there could be a difference if the class has a `initializer_list` constructor, in which case it might be called with an empty one, instead of the constructor with no arguments (not 100% sure). Any well behaved class should not distinguish between those two cases though.